cdc/processor/pipeline/table_actor.go

```diff
@@ -159,6 +159,11 @@ func NewTableActor(cdcCtx cdcContext.Context,
 	return table, nil
 }
 
+// Close implements Actor interface.
+// TODO: implements table actor stop here.
+func (t *tableActor) Close() {
```

Comment:
PTAL @sdojjy

Reply:
ok, I will fire a pr after this is merged